Software/Development/SemesterProject/System/systemInit.c

```diff
@@ -1,202 +1,202 @@
-//###########################################################################
```

Comment:
Is dangerous as fuck that you keep changing the whole file!!!!

Reply:
I don't understand why this is happening, i dont think i changed anything more than the flashing comments.

Reply:
Should we just go for it or how do you suggest to check if more changes were made?